project/ProtocolProject.scala

```diff
@@ -0,0 +1,59 @@
+import sbt._
+import sbt.Keys._
+import sbt.internal.util.ManagedLogger
+import sbtprotoc.ProtocPlugin
+import scala.sys.process._
+
+/**
+ * Settings for protocol projects.
+ */
+object ProtocolProject extends AutoPlugin {
```

Comment:
Picked from the framework project. After we release a new SDK we can also copy the plugin part that checks for breaking changes. I will ticketify this so we don't forget.